UnitTestBot/UTBotJava
# Fix problem with Pycharm plugin

## Description

Added a new module for Pycharm plugin `utbot-python-pycharm`. When you set `ideType=PC/PY` this module will be activated and `utbot-intellij-main` will be deactivated.

Fixes #2720

## How to test

### Manual tests

1. IDEA
  1.1. Set `ideType=IC` and run `:utbot-intellij-main:runIde`
  1.2.1. Open Java project and try to generate tests
  1.2.2. __Expected__: Tests have been generated
  1.3.1. Open Python project and try to generate tests
  1.3.2. __Expected__: Tests have been generated
  1.3.3. Open Utbot plugin in settings and check that it names "UnitTestBot"
2. Pycharm
  2.1. Set `ideType=PC` and run `:utbot-python-pycharm:runIde`
  2.2.1. Open Java project and try to generate tests
  2.2.2. __Expected__: You cannot generate tests
  2.3.1. Open Python project and try to generate tests
  2.3.2. __Expected__: Tests have been generated
  2.3.3. Open Utbot plugin in settings and check that it names "UnitTestBot for Python"


## Self-check list

Check off the item if the statement is true. Hint: [x] is a marked item.

Please do not delete the list or its items.
